arcgis-ios-sdk-samples/Features/Display feature layers/DisplayFeatureLayersViewController.swift

```diff
@@ -0,0 +1,183 @@
+        
+        alertController.popoverPresentationController?.barButtonItem = changeFeatureLayerBarButtonItem
+        present(alertController, animated: true)
+    }
+    
+    // MARK: Helper functions
+    
+    /// Load a feature layer with a URL.
+    func loadFeatureServiceURL() {
+        // Initialize the service feature table using a URL.
+        let featureTable = AGSServiceFeatureTable(url: URL(string: "https://sampleserver7.arcgisonline.com/server/rest/services/DamageAssessment/FeatureServer/0")!)
+        // Create a feature layer with the feature table.
+        let featureLayer = AGSFeatureLayer(featureTable: featureTable)
+        // Set the viewpoint to the Los Angeles National Forest.
+        let viewpoint = AGSViewpoint(latitude: 41.773519, longitude: -88.143104, scale: 4e3)
+        setMap(featureLayer: featureLayer, viewpoint: viewpoint)
+    }
+    
+    /// Load a feature layer with a portal item.
+    func loadPortalItem() {
+        // Set the portal.
+        let portal = AGSPortal.arcGISOnline(withLoginRequired: false)
+        // Create the portal item with the item ID for the Portland tree service data.
+        let item = AGSPortalItem(portal: portal, itemID: "1759fd3e8a324358a0c58d9a687a8578")
+        // Create the feature layer with the item and layer ID.
+        let featureLayer = AGSFeatureLayer(item: item, layerID: 0)
+        // Set the viewpoint to Portland, Oregon.
+        let viewpoint = AGSViewpoint(latitude: 45.5266, longitude: -122.6219, scale: 6e3)
+        setMap(featureLayer: featureLayer, viewpoint: viewpoint)
+    }
+    
+    /// Load a feature layer with a local geodatabase.
+    func loadGeodatabase() {
+        // Instantiate the geodatabase with the file name.
+        geodatabase = AGSGeodatabase(name: "LA_Trails")
+        
+        // Load the geodatabase for feature tables.
+        geodatabase.load { [weak self] (error: Error?) in
+            guard let self = self else { return }
+            if let error = error {
+                self.presentAlert(error: error)
+            } else {
+                // Get the feature table with the name.
+                let featureTable = self.geodatabase.geodatabaseFeatureTable(withName: "Trailheads")!
+                // Create a feature layer with the feature table.
+                let featureLayer = AGSFeatureLayer(featureTable: featureTable)
+                // Set the viewpoint to Malibu, California.
+                let viewpoint = AGSViewpoint(latitude: 34.0772, longitude: -118.7989, scale: 6e5)
+                self.setMap(featureLayer: featureLayer, viewpoint: viewpoint)
+            }
+        }
+    }
+    
+    /// Load a feature layer with a local geopackage.
+    func loadGeopackage() {
+        // Create a geopackage from a named bundle resource.
+        geoPackage = AGSGeoPackage(name: "AuroraCO")
+        
+        // Load the geopackage.
+        geoPackage?.load { [weak self] (error: Error?) in
```

Comment:
```suggestion
        geoPackage.load { [weak self] (error: Error?) in
```

in addition to https://github.com/Esri/arcgis-runtime-samples-ios/pull/1134#discussion_r785238736